emcc.py

```diff
@@ -3493,7 +3493,20 @@ def parse_string_list(text):
       return []
     return parse_string_list_members(inner)
 
+  def parse_json_list(text):
+    # if json parsing fails, we fall back to our own parser, which can handle a few
+    # simpler syntaxes
+    try:
+      return json.loads(text)
+    except:
+      return parse_string_list(text)
+
   if text[0] == '[':
+    # proper json can like ["a", "b"] can be simply handled by the python json parser
+    if text[1] == '"':
```

Comment:
Do we need to check this? The JSON would fail to parse `[a, b]`, and would fall back to `parse_string_list`.

Reply:
True, but it may be faster to avoid even entering the JSON code unnecessarily, and I also think it makes the code clearer as to what we can parse in this path and what we can't.

Reply:
Fair enough, I refactored it now following your suggestion and @quantum5's to avoid that check, which also lets us handle more json (including whitespace after the `[`).